index.js

```diff
@@ -1,29 +1,16 @@
-var os = require('os');
-var childProcess = require('child_process');
-var pkgInfo = require('./package.json');
+const os = require('os');
+const pkgInfo = require('./package.json');
+const SentryCli = require('./js/cli');
 
-var path = null;
+exports.default = SentryCli;
```

Comment:
Please use this instead:

```js
module.exports = SentryCli;
```

It is yet unusual in Node to use the ES6 style modules, especially in older versions.